trie/rocksdb/Cargo.toml

```diff
@@ -13,4 +13,4 @@ name = "trie_rocksdb"
 [dependencies]
 etcommon-trie = { version = "0.4", path = ".." }
 etcommon-bigint = { version = "0.2", path = "../../bigint" }
-rocksdb = { git = "https://github.com/paritytech/rust-rocksdb" }
\ No newline at end of file
+parity-rocksdb = { git = "https://github.com/paritytech/rust-rocksdb" }
```

Comment:
@elichai please use `crates.io` version of crate. Or are there any pitfalls?

Reply:
So there are no tests in this crate. not sure if anything even uses that, I changed it anyway because they're the same right now and it's better to be on a real version number and not on master anyway.